alacritty/src/input.rs

```diff
@@ -1272,6 +1294,24 @@ mod tests {
             window_id: unsafe { std::mem::transmute_copy(&0) },
         },
         end_state: ClickState::TripleClick,
+        end_button: MouseButton::Left,
+    }
+
+    test_clickstate! {
+        name: triple_click_middle,
+        initial_state: ClickState::DoubleClick,
+        initial_button: MouseButton::Middle,
+        input: GlutinEvent::WindowEvent {
+            event: WindowEvent::MouseInput {
+                state: ElementState::Pressed,
+                button: MouseButton::Middle,
+                device_id: unsafe { std::mem::transmute_copy(&0) },
+                modifiers: ModifiersState::default(),
+            },
+            window_id: unsafe { std::mem::transmute_copy(&0) },
+        },
+        end_state: ClickState::TripleClick,
+        end_button: MouseButton::Middle,
     }
```

Comment:
What's the benefit of the two new tests?

Reply:
These show that the double- and triple-click logic both work for buttons other than the left mouse button.  It seemed relevant since we rely on them working for the right-click expand selection so it picks the correct selection mode.  I can remove them if you don't think they are useful.

Reply:
I find these tests a bit questionable, while it makes some sense to test that buttons other than LMB work, it just seems like there's not a clear enough goal just throwing out a bunch of tests because it's easy.

The `end_button` especially is somewhat questionable, since there's really no good reason I see why that would change or be relevant.

So I think I'd rather just skip these tests, then the PR should be good.